src/message.coffee

```diff
@@ -87,12 +87,12 @@ class SlackTextMessage extends TextMessage
   ###
   buildText: (client, cb) ->
     # base text
-    text = @rawMessage.text
+    text = if @rawMessage.text? then @rawMessage.text else ""
 
     # flatten any attachments into text
     if @rawMessage.attachments
       attachment_text = @rawMessage.attachments.map((a) -> a.fallback).join("\n")
-      text = text + "\n" + attachment_text
+      text = text + "\n" + attachment_text if attachment_text.length > 0
```

Comment:
Might be simpler to use `if !!attachment_text` here

Reply:
i personally think this is more explicit and nicer. `attachment_text` is always going to be a string (thanks to `Array.prototype.join()`), its just that sometimes it will be an empty string. there's no other falsy value to be concerned with.

Reply:
👍